pkg/antctl/antctl.go

```diff
@@ -129,12 +129,17 @@ var CommandList = &commandList{
 					params: []flagInfo{
 						{
 							name:  "name",
-							usage: "Get NetworkPolicy by name. If present, Namespace must be provided.",
+							usage: "Get NetworkPolicy by name.",
 							arg:   true,
 						},
+						{
+							name:      "source",
+							usage:     "Get Networkpolicies with specific source. Source means the name of its source network policy.",
```

Comment:
Maybe clarify the meaning of source a bit better for our users: "The source of a computed NetworkPolicy is the original policy resource (K8s NetworkPolicy or Antrea-native Policy) from which the computed NetworkPolicy was derived."

Reply:
Done, thanks.